### ISSUE TITLE
When the `h1` is within the `legend` on the question component,  JAWS fails to find the `h1` when navigating by headings.

### ISSUE THREAD

**Issue by bameyrick** [ONSdigital/design-system#450]
When the `h1` is within the `legend`,  JAWS fails to find the `h1` when navigating by headings.

_Originally posted by @bameyrick in https://github.com/ONSdigital/design-system/issues/367#issuecomment-503565087_
